src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/BlobManager.java

```diff
@@ -222,4 +242,16 @@ private void logAcquireLeaseError(String zipFilename, String containerName, Exce
             exception
         );
     }
+
+    private boolean readyToAcquireLease(CloudBlockBlob cloudBlockBlob) {
+        String leaseAcquiredAt = cloudBlockBlob.getMetadata().get(LEASE_ACQUIRED_TIME);
+        if (StringUtils.isBlank(leaseAcquiredAt)) {
+            return true;
+        } else {
+            LocalDateTime leaseAcquiredAtTime = LocalDateTime.parse(leaseAcquiredAt);
+            Duration timeDifference = Duration.between(LocalDateTime.now(), leaseAcquiredAtTime);
```

Comment:
should `now` also include `EUROPE_LONDON_ZONE_ID` here too?

Reply:
yeah, will update